java/mykola-melnyk/src/com/zoolatech/lecture4/tasks/_7/DigitToString.java

```diff
@@ -0,0 +1,32 @@
+package com.zoolatech.lecture4.tasks._7;
+
+import java.util.Arrays;
+
+public class DigitToString {
+
+    public static String convertToWord(char digit) {
+        String word = null;
+        switch (digit) {
```

Comment:
same: you can use the switch expression here

Reply:
Done